api/src/main.rs

```diff
@@ -23,6 +23,14 @@ pub async fn main() -> anyhow::Result<()> {
             info!("{configuration}");
             Application::migrate_database(configuration).await?;
             info!("database migrated successfullly");
+        } else if command == "delete-test-databases" {
```

Comment:
Is it possible to drop non-test databases by this command?

Reply:
This command drops all databases which are not named `postgres`, `template0`, or `template1`. That is ok because it is only for running on the local test db.

Reply:
Since all test dbs have names as randomly generated uuids, to further reduce the risk of non-test databases being deleted accidentally, added a filter to only delete databases with names as valid uuids.